### DISCUSSION TITLE
nonebot.log 模块如何设置error级别的文件日志

### DISCUSSION THREAD

**Discussion by H-Elden** [nonebot/discussions#546] [问答] [answered]
运行环境：
- windows 11
- conda虚拟环境
- python=3.10.11

我使用的是nb脚手架创建的 nonebot 项目，选择的是提供的simple模板，也就是说没有 `bot.py` 入口文件，只能通过 `nb run` 指令启动程序。

需求：仍然使用stdout在终端输出INFO级别的日志信息，在此基础上将所有的ERROR级别的日志信息写入error.log文件。

官方文档(https://nonebot.dev/docs/appendices/log)  只给出了在有入口文件的时候，可以在 `nonebot.init` 之前添加 `logger.add` 方法来实现。但是没有提到在没有入口文件的情况下的使用方法。

**Comment by yanyongyu** [comment, marked as answer]
没有入口文件不建议修改

**Reply by H-Elden** [reply]
好吧，那么我只好使用入口文件的方式修改我的程序了。感谢解答！
